docs/releasenotes.md

```diff
@@ -1,5 +1,12 @@
 # Releasenotes Haal-Centraal-BRP-Bevragen
 
+## Versie 1.2.0
+
+- De property "adellijkeTitelPredikaat" is verplaatst van "NaamPersoon" naar "Naam"
+  Dit heeft als gevolg dat deze property nu ook onderdeel is van de resources "ouder", "kind" en "partner"
+- De property "adellijkeTitelPredikaat" (boolean) is toegevoegd aan "InOnderzoek" van het schemacomponent "Naam"
+- Bij "NaamPersoon" is de boolean "aanduidingNaamGebruik" toegevoegd aan "Inonderzoek"  
```

Comment:
"Inonderzoek" moet dat niet zijn "inOnderzoek" (propertynaam)  of "NaamInOnderzoek" (componentnaam)?

Reply:
Aangepast